wiki/en/Running-a-Server.md

```diff
@@ -52,7 +52,7 @@ Unless you plan on hosting more than about 5 players on a slower-speed home conn
 
 - Running a Private Server at home will require you to [port forward](#running-a-private-server) on your router. When running a Private Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
 
-- Jamulus doesn't currently support IPv6
+- Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the client and server. There are plans to expand IPv6 support for connections started from the server list.
```

Comment:
There's more to it than that.  I'd rather not over-simplify things just for the sake of more words.

Reply:
Yes, it’s more than that. 
@gilgongo should we revert my proposal then?

Reply:
OK sure, it's only really a half-feature at the moment and we don't want to make a big thing about it yet.

Reply:
`Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the client and server.` alone, would do for now.  _Maybe_ say "Directory servers are not supported" (which would cover server registration with a directory server over IPv6 and running a directory server on IPv6).